pkg/gitops_test.go

```diff
@@ -2131,3 +2187,29 @@ func getCommitIDFromDotGit(repoPath string) (string, error) {
 	}
 	return string(fileBytes), nil
 }
+
+func mockExecute(outputStack *testutils.OutputStack, errorStack *testutils.ErrorStack, executedCmds *[]testutils.Execution, baseDir string, cmd CommandType, args ...string) ([]byte, error, *[]testutils.Execution) {
```

Comment:
So each test 'file' we have will have a similar mockExecute?

Reply:
> So each test 'file' we have will have a similar mockExecute?

Only if you are testing a function that runs a `git` or `rm` execute command (and you want to avoid live calls).  If the file is in the same package and the current mockExecute behaviour is sufficient, you can just call it, otherwise you will need to create your own.